.gitmodules

```diff
@@ -0,0 +1,9 @@
+[submodule "themes/ataraxia"]
+	path = themes/ataraxia
+	url = https://github.com/gersonbenavides/ataraxia-zola.git
+[submodule "static/assets"]
+	path = static/assets
+	url = https://github.com/rustdelhi/assets.git
+[submodule "docs/assets"]
```

Comment:
`zola build` command should prepare the `docs` dir for us. This may not be needed.

Reply:
Previously i had the submodule in the static dir only but the zola build copies static/assets/.git dir so git assumes that docs/assets is a submodule

I've also tried adding docs/assets/.git to gitignore but it doesn't work and the only workaround was to add it as submodule in docs dir as well.

Reply:
got it. so removing the docs altogether folder would be the simple solution to this. considering this issue resolved for now. we can setup a github workflow to build in a future PR.